Gemfile

```diff
@@ -32,6 +31,15 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+# Use omniauth for session handling
+gem 'omniauth'
```

Comment:
Both removed and then added again per common instructions from Omniauth strategy authors.